### ISSUE TITLE
How to call env.render(mode='rgb_array') without a window popup?

### ISSUE THREAD

**Issue by schissmantics** [openai/gym#2351]
**Note:** Due to another inappropriate closing of an unresolved [issue](https://github.com/openai/gym/issues/2349) that was conducted earlier  by some [user](https://github.com/jkterry1), I'm re-opening the issue. 

I need to the following on macos `Big Sur 11.4`, `python3.9.5`, `gym==0.18.3` to get frame as an array which is not returned by default for bipedal walker env.

    import gym
    
    
    env = gym.make('BipedalWalker-v3')
    state = env.render(mode='rgb_array')

This does the job however, I don't want a window popping up because this will be called by pytest so, that window beside requiring a virtual display if the tests are run remotely on some server, is unnecessary. Is there a way to prevent the window from popping up?

**Comment by mickelliu** [comment]
I think this is a useful feature. I was using Rllib `render_env` feature (which is done through `gym.monitor`) and I found it quite annoying to have 30 viewers popped up that must remain open while recording videos.

**Comment by schissmantics** [comment]
Yeah, I think `gym.wrappers.Monitor` does the same thing. It keeps populating the screen with those windows which are impossible to close during runtime. I tried some workarounds but the solutions that work fix the problem and create other problems.
